main.go

```diff
@@ -225,34 +200,40 @@ func main() {
+	filterHelp += "Examples:\n"
+	filterHelp += "\t--filter event=execve,open                                    | only trace execve and open events\n"
+	filterHelp += "\t--filter set=fs                                               | trace all file-system related events\n"
+	filterHelp += "\t--filter s=fs --filter e!=open,openat                         | trace all file-system related events, but not open(at)\n"
+	filterHelp += "\t--filter uts!=ab356bc4dd554                                   | don't trace events from uts name ab356bc4dd554\n"
+	filterHelp += "\t--filter comm=ls                                              | only trace events from ls command\n"
 	filterHelp += "\n"
-	filterHelp += "mntns: only trace processes or containers with specified mount namespace(s) ids.\n"
-	filterHelp += "\t--filter mntns=12345678                                       | only trace events from mntns 12345678\n"
-	filterHelp += "\t--filter mntns!=12345678                                      | don't trace events from mntns 12345678\n"
+	filterHelp += "boolean filters use the following operator: '!'\n"
+	filterHelp += "available boolean filters: container\n"
+	filterHelp += "Examples:\n"
+	filterHelp += "\t--filter container                                            | only trace events from containers\n"
+	filterHelp += "\t--filter c                                                    | only trace events from containers (same as above)\n"
+	filterHelp += "\t--filter !container                                           | only trace events from the host\n"
+	filterHelp += "\t--filter '!c'                                                 | only trace events from the host (same as above)\n"
 	filterHelp += "\n"
-	filterHelp += "pidns: only trace processes or containers with specified pid namespace(s) ids.\n"
-	filterHelp += "\t--filter pidns=12345678                                       | only trace events from pidns 12345678\n"
-	filterHelp += "\t--filter pidns!=12345678                                      | don't trace events from pidns 12345678\n"
-	filterHelp += "uts: only trace processes or containers with specified uts namespace(s) name.\n"
-	filterHelp += "\t--filter uts=8215606f23f4                                     | only trace events from uts 8215606f23f4\n"
-	filterHelp += "\t--filter uts!=ab356bc4dd554                                   | don't trace events from uts ab356bc4dd554\n"
-	filterHelp += "comm: only trace processes with specified command name.\n"
-	filterHelp += "\t--filter comm=ls                                              | only trace events from ls command\n"
-	filterHelp += "\t--filter comm!=ls                                             | don't trace events from ls command\n"
+	filterHelp += "Note: some of the above operators have special meanings in different shells\n"
+	filterHelp += "      to 'escape' those operators, please use single quotes, e.g.: 'uid>0'\n"
```

Comment:
can you please replace these string constructions with a raw string? (between `)

Reply:
You mean string builder, right?

Reply:
no. theres not string to build here at runtime, it's just one big string in the source code.

```go
s:= `this is 
a raw string`
```